internal/backend/sessions/sessionworkflows/syscalls.go

```diff
@@ -198,8 +198,9 @@ Get the next event on this signal (has to exists since we got a signal on it)
 return this event
 */
 func (w *sessionWorkflow) nextEvent(ctx context.Context, args []sdktypes.Value, kwargs map[string]sdktypes.Value) (sdktypes.Value, error) {
-	if len(kwargs) > 0 {
-		return sdktypes.InvalidValue, errors.New("unexpected keyword arguments")
+	var timeout time.Duration
```

Comment:
time.Duration is int64, it can be negative
I think we should fail on negative value or max it to 0
```go
timeout = max(0, timeout)
```

Reply:
i think we should error out if < 0. added check.